test/check-application

```diff
@@ -1913,6 +1913,40 @@ class TestApplication(testlib.MachineCase):
         self.performContainerAction(container_name, "Resume")
         b.wait(lambda: self.getContainerAttr(container_name, "State") == "Running")
 
+    def testRenameContainerSystem(self):
+        self._testRenameContainer(True)
+
+    def testRenameContainerUser(self):
+        self._testRenameContainer(False)
+
+    def _testRenameContainer(self, auth):
+        b = self.browser
+        container_name = "rename"
+        container_name_new = "rename-new"
+
+        self.execute(auth, f"podman container create -t --name {container_name} quay.io/libpod/busybox:latest")
+        self.login(auth)
+
+        self.filter_containers('all')
+
+        self.waitContainerRow(container_name)
+        self.toggleExpandedContainer(container_name)
+        self.performContainerAction(container_name, "Rename")
+
+        # the container name should be in the "Rename container" header
+        b.wait_in_text("#pf-modal-part-1", container_name)
+
+        b.set_input_text("#rename-dialog-container-name", container_name_new)
+        b.click('#btn-rename-dialog-container')
+        b.wait_not_present("#rename-dialog-container-name")
+
+        self.execute(auth, f"podman inspect --format '{{{{.Id}}}}' {container_name_new}").strip()
+        self.waitContainerRow(container_name_new)
+
+        # test if rename is hidden for running container. This should change with rename events fixed in Podman.
```

Comment:
Getting an event for a rename seems independent from supporting renames for running containers. Is that comment actually true? I thought events were already fixed in podman 4.1.

Reply:
I did some digging, and indeed the rename event is added in 4.1 https://github.com/containers/podman/releases/tag/v4.1.0

About renaming a running container, this works surprisingly well?

```
[root@fedora-36-127-0-0-2-2201 ~]# podman ps
CONTAINER ID  IMAGE                          COMMAND     CREATED       STATUS            PORTS       NAMES
359ac580010e  quay.io/libpod/busybox:latest  /bin/sh     1 second ago  Up 2 seconds ago              foo
[root@fedora-36-127-0-0-2-2201 ~]# podman rename foo bar
[root@fedora-36-127-0-0-2-2201 ~]# podman ps
CONTAINER ID  IMAGE                          COMMAND     CREATED        STATUS             PORTS       NAMES
359ac580010e  quay.io/libpod/busybox:latest  /bin/sh     9 seconds ago  Up 10 seconds ago              bar
[root@fedora-36-127-0-0-2-2201 ~]# podman -v
podman version 4.0.2
```

Reply:
And also on 3.0.1?

```
root@debian:~# podman ps
CONTAINER ID  IMAGE    COMMAND  CREATED       STATUS            PORTS   NAMES
cdc59dfb3e96  busybox  /bin/sh  1 second ago  Up 2 seconds ago          foo
root@debian:~# podman rename foo bar
root@debian:~# podman ps
CONTAINER ID  IMAGE    COMMAND  CREATED        STATUS            PORTS   NAMES
cdc59dfb3e96  busybox  /bin/sh  7 seconds ago  Up 7 seconds ago          bar
root@debian:~# podman -v
podman version 3.0.1
```